README.md

```diff
@@ -181,7 +181,7 @@ jobs:
       - name: actions-template-sync
         uses: AndreasAugustin/actions-template-sync@v2
         with:
-          org_github_token: ${{ secrets.ORG_GITHUB_TOKEN }}
```

Comment:
the secret possibly can be renamed within the docs. Right. Need to think about it

Reply:
Maybe source_org_github_token github token would be a better name but this assumes that you are working within an organization.

Reply:
thanks for the suggestion. Yes this has the assumption that you work within an org. Furthermore there are other AUTH tokens available like a PAT. Therefore I think using `org` is not generic enough.